packages/data/walkthroughs/9.9.9.json

```diff
@@ -1982,22 +2012,24 @@
     },
     "P21": {
       "walkthroughItemType": "multiChoice",
-      "questionText": "The egress door of the dwelling unit opens onto ...",
+      "questionText": "The egress door of the <defined-term>dwelling unit</defined-term> opens onto ...",
       "possibleAnswers": [
         {
-          "answerDisplayText": "An exit stairway",
+          "answerDisplayText": "An <defined-term>exit</defined-term> stairway",
           "answerValue": "A"
         },
         {
-          "answerDisplayText": "A public corridor",
+          "answerDisplayText": "A <defined-term>public corridor</defined-term>",
           "answerValue": "B"
         },
         {
           "answerDisplayText": "An exterior passageway",
+          "answerValueDisplay": "exterior passageway",
```

Comment:
For my understanding, what's an `answerValueDisplay`?

Reply:
That is the display value for when that answer is used later in a question (for the question text fill in for P23 and P24). I thought about just making the `answerValue` also a display appropriate value but wasn't sure about a case where we'd want them different so opted for this. Thoughts?

Reply:
Makes sense to me.